app/assets/stylesheets/frontend/views/publications/_show.scss

```diff
@@ -0,0 +1,120 @@
+.publications-show-page {
+  p {
+    @include core-19;
+  }
+  .govspeak ul {
+    margin-left:0px;
+  }
+
+  .heading-block {
+    h1 {
+      @include media(tablet) {
+        padding-left: $gutter-half;
+        margin-bottom: $gutter-two-thirds;
+      }
+    }
+    .document-type {
+      @include core-19;
+      float: left;
+      margin-right: 5px;
+      @include media(tablet) {
+        @include bold-24;
```

Comment:
I don't think that doing fonts this way (specifying our mixins within media blocks) is a good idea. At first read I can't work out what font size will be in effect where and how the media queries are likely to join together.